### ISSUE TITLE
Need margin

### ISSUE THREAD

**Issue by karino2** [karino2/kotlitex#95]
Currently, no margin for expression and it's layout-ed too tightly.
I think hard coded margin is suffice for current stage.

**Comment by kzys** [comment]
The height is also too tight. 

<img width="445" alt="Untitled" src="https://user-images.githubusercontent.com/19111/55285495-2712c800-5342-11e9-8125-96eb3d79de92.png">

**Comment by kzys** [comment]
From MathExpressionSpan;

```
            // val lineNum = rect.bottom/BASE_HEIGHT
            // We follow DynamicDrawableSpan logic. But it's a little different from no-size specified case.
            // fm.ascent = -(lineNum*targetHeight).roundToInt()
            /*
            fm.ascent = -(rect.bottom*ratio).roundToInt()
            fm.descent = 0
            fm.top = fm.ascent
            fm.bottom = 0
            */

            fm.ascent = -ascent
            fm.descent = descent

            fm.bottom = fm.descent
            fm.top = fm.ascent
```

Actually DynamicDrawableSpan's logic (fm.ascent = -rect.bottom, fm.top = fm.ascent, and the rest is zero) works better in my environment. What was the reason of doing the above?

**Comment by karino2** [comment]
I thought this code is better, but I try now and seems original code works better as you said.
I'll modify to original code. Sorry.

**Comment by karino2** [comment]
Previous comment was wrong.
There IS a reason.

For wrapping case, original code wrongly layout expression.
Sigma overdraw "text".

![15540268242570](https://user-images.githubusercontent.com/1468643/55287771-58de6a00-53e8-11e9-87c1-4b4e337c1327.jpg)

And current code seems better.

![15540268525591](https://user-images.githubusercontent.com/1468643/55287776-6562c280-53e8-11e9-80b0-255a3ffd5f2f.jpg)

But this code is also wrong because top is not within correct region.

I guess later one is closer to real answer, but may be not.

**Comment by karino2** [comment]
This was #94.

**Comment by karino2** [comment]
Now I modify top value to original one.
It seems layout result is correct, but I don't know why.

**Comment by karino2** [comment]
Like this.

![NormalAppImage(8)](https://user-images.githubusercontent.com/1468643/55462727-5bcc9b00-5632-11e9-8517-e3b6deefa0b4.png)

In device, it's too close (though in this image, it's not so bad).

**Comment by kzys** [comment]
You meant that `2` and `n/2` is too close, or `2^(n/2)` and the text is too close?

**Comment by karino2** [comment]
I mean between 2^(n/2) and the text.

**Comment by karino2** [comment]
Now fixed by ad-hock manner in #163 .
